README.md

```diff
@@ -1,2 +1,63 @@
-# template-public
-general template for public repositories
+# tfc-backup-b2
+Docker image to export variables from Terraform Cloud and back them up to a
+Restic repository on Backblaze B2.
+The image can also initialize the Restic repository on the existing
+Backblaze B2 bucket.
+
+## Description
+During the review of a disaster recovery plan, we realized that we didn't have
+a record of the values we set for variables in Terraform Cloud workspaces.
+It would be difficult to recover from the accidental deletion of a Terraform
+Cloud workspace.
+A Perl script exports workspaces, variables, and variable sets to JSON files
+using the Terraform Cloud API.
+The JSON files are then backed up using Restic to a repository on a Backblaze
+B2 bucket.
+
+Two files are created for each Terraform Cloud workspace:
+
+- _workspace-name_-workspace.json
```

Comment:
Optional suggestion:
```suggestion
- _workspace-name_-attributes.json
```

Reply:
Agreed.  Also changed varset-_variable-set-name_.json to varset-_variable-set-name_-attributes.json.